docker/container.go

```diff
@@ -327,7 +324,7 @@ func (c *Container) Log(ctx context.Context, l logger.Logger, follow bool) error
 		Follow:     follow,
 		Tail:       "all",
 	}
-	responseBody, err := c.client.ContainerLogs(ctx, c.name, options)
+	responseBody, err := c.client.ContainerLogs(ctx, c.container.Name, options)
```

Comment:
@joshwget oh you're right. I changed to `c.ID` in another PR in preparation, I should do it here :angel:. I'll update :wink: